test/controllers/browse_controller_test.rb

```diff
@@ -60,33 +60,27 @@
 
       describe "with the new navigation enabled" do
         it "returns the original version of the page as the A variant" do
-          with_new_navigation_enabled do
-            with_variant EducationNavigation: "A" do
-              get :show, top_level_slug: "education"
+          with_new_navigation_disabled do
+            get :show, top_level_slug: "education"
 
-              assert_response 200
-            end
+            assert_response 200
           end
         end
 
         it "redirects to the taxonomy navigation as the B variant" do
-          with_new_navigation_enabled do
-            with_variant EducationNavigation: "B", assert_meta_tag: false do
-              get :show, top_level_slug: "education"
+          with_new_navigation_enabled assert_meta_tag: false do
+            get :show, top_level_slug: "education"
 
-              assert_response 302
-              assert_redirected_to controller: "taxons", action: "show", taxon_base_path: "education"
-            end
+            assert_response 302
+            assert_redirected_to controller: "taxons", action: "show", taxon_base_path: "education"
           end
         end
 
         ["A", "B"].each do |variant|
           it "does not change a page outside the A/B test when the #{variant} variant is requested" do
             setup_ab_variant("EducationNavigation", variant)
 
-            with_new_navigation_enabled do
```

Comment:
Is this test still testing the right thing? It looks like it no longer sets the environment variable, so it would pass even if the `get` request was to an education page, which isn't right.

Same goes for the JSON format test below.